lua/neotest-kotlin/kotest-treesitter-query.lua

```diff
@@ -1,46 +1,39 @@
-local TreesitterQuery = {}
-
--- These are treesitter queries for pulling data of of the AST,
+-- These are treesitter queries for pulling data out of the AST,
 -- More information on how this works over here: https://neovim.io/doc/user/treesitter.html
--- And you can interactively play around with these kotlin queries here: https://fwcd.dev/tree-sitter-kotlin/
-
-TreesitterQuery.value = [[
```

Comment:
Removed this wrapping in a table, to just set one property of the table to just return the String itself. This was being done in the treesitter/*.lua files by directly setting M equal to the String